views/results/cache.html

```diff
@@ -0,0 +1,8 @@
+{{ if .pageCache }}
+    {{ str2html .pageCache }}
+{{ else }}
+    We cannot find the result you are looking for
+    <a href="/">
+        Back to Home
+    </a>
```

Comment:
Can we move this as a partial? Because we use this in both `cache.html` and `show.html`

Reply:
Yeah actually I moved them to a partial on [another PR](https://github.com/carryall/go-google-scraper-challenge/pull/73) 😄